src/HomePage/components/Search/Search.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import "./SearchStyles.css"
+
+export function Search(props: { onSearchMovie: (arg: string) => void; }) {
+    const handleChange = (e: any) => {
+        props.onSearchMovie(e.target.value);
+    };
```

Comment:
NIT: Good for you for trying to add on search handler. But please whenever you add something like this (handler for user typing), consider using debounce to prevent unnecessary calculations, rerendering and changes.